### PR TITLE
automl-text-classification

### PR DESCRIPTION
**Pull request by udaypunna**
**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Changed according to boilerplate requirement.
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/automl/automl-text-classification.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "cells": [
```
Comment (ivanmkc) [inline notebooks/official/automl/automl-text-classification.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #3.    display_name = f'{"e2e-text-dataset-unique"}'
</pre><p>Remove curly braces and f.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Comment (ivanmkc) [inline notebooks/official/automl/automl-text-classification.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #2.    training_job_display_name = f'{"e2e-text-training-job-unique"}'
</pre><p>Remove curly braces and f.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (ivanmkc) [inline notebooks/official/automl/automl-text-classification.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    model_display_name = f'{"e2e-text-classification-model-unique"}'
</pre><p>Remove curly braces and f.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (ivanmkc) [inline notebooks/official/automl/automl-text-classification.ipynb:2]: <p>Change this to "Review results"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1244/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Comment (udaypunna) [reply notebooks/official/automl/automl-text-classification.ipynb:2]: <p>Remove curly braces and f.</p>

Comment (udaypunna) [reply notebooks/official/automl/automl-text-classification.ipynb:2]: <p>Remove curly braces and f.</p>

**Review by ivanmkc — APPROVED** [review]
LGTM

**Pull request merged**